lasagne/nonlinearities.py

```diff
@@ -22,10 +82,31 @@ def rectify(x):
 
 
 # leaky rectify
-# Maas et al: Rectifier Nonlinearities Improve Neural Network Acoustic Models
-# http://web.stanford.edu/~awni/papers/relu_hybrid_icml2013_final.pdf
 class LeakyRectify(object):
+    """Implementation of a leaky rectifier
+    :math:`\\varphi(x) = \max(leakiness \cdot x, x)`.
+
+    Parameters
+    ----------
+    leakiness : float in [0, 1]
```

Comment:
Thinking again, the range is not needed here. Values outside of that range work fine. I'd just prepend a sentence to the description below: "Slope of the function for :math:`x < 0`."